src/assets/scripts/client/aircraft/AircraftModel.js

```diff
@@ -1753,9 +1753,12 @@ export default class AircraftModel {
         const headingDifference = angle_offset(course, this.heading);
         const bearingFromAircaftToRunway = this.positionModel.bearingToPosition(datum);
         const angleAwayFromLocalizer = course - bearingFromAircaftToRunway;
-        const turnTimeInSeconds = abs(headingDifference) / PERFORMANCE.TURN_RATE;    // time to turn headingDifference degrees
+        const turnTimeInMilliSeconds = abs(headingDifference) / PERFORMANCE.TURN_RATE * TIME.ONE_SECOND_IN_MILLISECONDS;    // time to turn headingDifference degrees
+        // Radians / seconds is standard and will be used as seconds. 
         // TODO: this should be moved to a class method `.getTurningRadius()`
-        const turningRadius = this.speed * (turnTimeInSeconds * TIME.ONE_SECOND_IN_HOURS);  // dist covered in the turn, nm
+        const turningRadius = this.speed * (turnTimeInMilliSeconds * TIME.ONE_HOUR_IN_MILLISECONDS);  // dist covered in the turn, nm
```

Comment:
Shouldnt it be `this.speed * (turnTimeInMilliSeconds * TIME.ONE_MILLISECOND_IN_HOURS)`?